src/user_interface/playing_page.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, output, song_name="DEFAULT"):
         ## vbox.addWidget()  ## ADD PROGRESS BAR WIDGET HERE to vbox
         ############################################################
 
-        vbox.addWidget(VisualizationWidget(parent=self, output=output))
+        #vbox.addWidget(VisualizationWidget(parent=self, output=output))
```

Comment:
Don't remove my visualizer!

Reply:
Going to add a todo for the IPC swtichover